tests/jvm/src/test/resources/metac-metacp.diff

```diff
@@ -252,23 +252,23 @@ flags.p.package.
        symlinks: "flags.p.package.xs1()."
 
 
-=====================
-flags.p.package.S#[T]
-=====================
+====================
+flags.p.package.m().
+====================
 --- metac
 +++ metacp
- kind: TYPE_PARAMETER
- properties: 0
- name: "T"
--annotations {
--  tpe {
--    typeRef {
--      prefix {
--      symbol: "scala.specialized#"
--}
- accessibility {
-   tag: PUBLIC
-   symbol: ""
+kind: MACRO
+properties: 0
+name: "m"
++annotations {
++  tpe {
++    typeRef {
++      prefix {
++      symbol: "scala.reflect.macros.internal.macroImpl#"
```

Comment:
I can't tell if it's synthetic or not via the [AttributeInfo](https://static.javadoc.io/org.scala-lang/scalap/2.12.6/scala/tools/scalap/scalax/rules/scalasig/AttributeInfo.html) as far as I know.

Reply:
Let's manually filter out `scala.reflect.macros.internal.macroImpl#` annotations.